cyoa/app/views/rounds/show.html.haml

```diff
@@ -0,0 +1,12 @@
+WHICH ONE DID RON SWANSON SAY?
+- @current_question.quotes.shuffle.each do |quote|
+  %p=quote.content
+  =form_for @current_question, data: {round_number: 'cats'} do |f|
+    =f.hidden_field :answer, value: quote.id
+    =f.submit value: "Ron Said This", data: {submit_answer: 'blah'}
+
+
+
+%p
+%div{data:{result_holder: 'display'}}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace